libnmstate/schemas/operational-state.yaml

```diff
@@ -408,3 +413,29 @@ definitions:
                     type: boolean
                   state:
                     type: string
+  route:
+    type: object
```

Comment:
This should not be here (it is already present on line 30). The same convention has been used with the others as well.